TradeItIosTicketSDK2/TradeItInstrumentOrderCapabilitiesHelpers.swift

```diff
@@ -46,12 +45,17 @@ extension TradeItInstrumentOrderCapabilities {
         }
     }
 
+    private func supportedOrderQuantityTypesFor(capabilities: [TradeItInstrumentCapability], value: String) -> [OrderQuantityType] {
```

Comment:
IMO cleaner if the method is named `supportedOrderQuantityTypes` and the labels are `forCapabilities` `andValue`